src/utils/format-chat-history.ts

```diff
@@ -100,7 +99,7 @@ async function createContextBlockSection(
 
   const diffBlock = [
     createHeader("Linked Pull Request Code Diff", repoString),
-    diff ? diff : "No diff available",
+    isPull ? isPull : "No diff available",
```

Comment:
If it is not a pull shouldn't this revert or throw? Doesn't make sense that it would treat an issue as a pull.

Reply:
just above it we return early if `!isPull` and we know it's a pull only if we can get a diff string from `octokit.pulls.get().data.diff`